files/fr/web/api/clipboard_api/index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: API Clipboard
+slug: Web/API/Clipboard_API
+translation_of: 'Web/API/Clipboard_API'
+---
+<div>{{DefaultAPISidebar("Clipboard API")}}</div>
+
+<p>l'<strong>API Clipboard</strong> (en français : API Presse-papiers) fournit la possibilité de répondre aux commandes du presse-papiers (couper, copier et coller) ainsi que de lire et écrire sur le presse-papiers système de façon asynchrone. L'accès aux contenus du presse-papiers est protégé par l'<a href="/fr/docs/Web/API/Permissions_API">API Permissions</a> : la permission <code>clipboard-write</code> est donnée automatiquement aux pages lorsqu'elles sont dans l'onglet actif. La permission <code>clipboard-read</code> doit quant à elle être demandée, ce que vous pouvez faire en tentant de lire les données du presse-papiers.</p>
```

Comment:
L in uppercase at the beginning :)

Reply:
I wanted to say something about presse-papier vs presse-papiers but hey, seems https://transvision.mozfr.org/?recherche=presse-papier&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings is OK with you and I won't stand a chance :)

Reply:
Ok :D 
The typo is fixed in https://github.com/mdn/translated-content/pull/1770/commits/006b650c77390ce8185d51fb09d2f3c0345f4bd2